scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1143,7 +1134,7 @@ public void removeThirdPartyCredential(String key) throws NotConnectedException,
             boolean running, boolean pending, boolean finished, int offset, int limit)
                     throws NotConnectedException, PermissionException {
         RestPageParameters params = new RestPageParameters(frontendState, "getTaskIds", from, to, mytasks,
-                running, pending, finished, offset, limit, taskTag);
+                running, pending, finished, offset, limit, taskTag, null);
```

Comment:
It would be cleaner from an OO point of view to pass an instance of `SortSpecifierContainer` that contains no sort item. This ways, several _null_ checks can be avoided. This special instance of SortSpecifierContainer could be cached in a static field that is publicly available.